### PR TITLE
Elaborate on how the tutorial differs from the real packaging process

### PR DESCRIPTION
**Pull request by alchemyst**
This addresses [this issue](https://github.com/pypa/packaging-problems/issues/246). It is meant to help people who want to use the tutorial as a scaffold for getting their own package onto PyPI to understand what steps are required and which ones are just for the tutorial.

### CODE REVIEW

File: source/tutorials/packaging-projects.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ Creating setup.py
 about your package (such as the name and version) as well as which code files
 to include.
 
-Open :file:`setup.py` and enter the following content. You **should** update the package name to include your username (for example, ``example-pkg-theacodes``. You can personalize the other values if you'd like:
+Open :file:`setup.py` and enter the following content. To avoid lots of projects with the same name from people doing this tutorial, you **should** update the package name to include your username (for example, ``example-pkg-theacodes``). You can personalize the other values if you'd like:
```
Comment (theacodes) [inline source/tutorials/packaging-projects.rst]: Maybe phrase this as `Update the package name to include your username (for example, ``example-pkg-theacodes``), this ensures that you have a unique package name and that your package doesn't conflict with packages uploaded by other people following this tutorial.`

Comment (alchemyst) [reply source/tutorials/packaging-projects.rst]: ✅

Comment (theacodes) [reply source/tutorials/packaging-projects.rst]: Hrm, I think you updated the wrong one. The one below should remain short (just a reminder of the instructions here).

Comment (alchemyst) [reply source/tutorials/packaging-projects.rst]: Sorry, should be fixed now.

**Pull request merged**